test/Microsoft.ML.Tests/TrainerEstimators/MatrixFactorizationTests.cs

```diff
@@ -54,7 +54,6 @@ public void MatrixFactorization_Estimator()
         }
 
         [MatrixFactorizationFact]
```

Comment:
Is MatrixFactorizationFact needed?

Reply:
Actually it is not! Currently, `MatrixFactorizationFact` is just an `EnvironmentSpecificFactAttribute` that always returns `true` with `IsEnvironmentSupported()`. In a seperate PR, I'll be marking these `MatrixFactorizationFact` marked tests as `Fact`, and if possible, remove the `MatrixFactorizationFact` from the codebase all together.

Reply:
this is already done in below PR: [https://github.com/dotnet/machinelearning/pull/5122](https://github.com/dotnet/machinelearning/pull/5122)


---
In reply to: [424743738](https://github.com/dotnet/machinelearning/pull/5121#discussion_r424743738) [](ancestors = 424743738)